### PR TITLE
Docblock cleanup

### PR DESCRIPTION
**Pull request by dereuromark**
Small cleanups

### CODE REVIEW

File: src/Propel/Generator/Builder/DataModelBuilder.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ abstract class DataModelBuilder
     /**
      * An array of warning messages that can be retrieved for display.
      *
-     * @var array string[]
+     * @var array<string>
```
Comment (gechetspr) [inline src/Propel/Generator/Builder/DataModelBuilder.php:57]: mb it should be `list` too?

File: src/Propel/Generator/Reverse/AbstractSchemaParser.php [hunk 2]
```diff
@@ -31,7 +31,7 @@ abstract class AbstractSchemaParser implements SchemaParserInterface
     /**
      * Stack of warnings.
      *
-     * @var array string[]
+     * @var array<string>
```
Comment (gechetspr) [inline src/Propel/Generator/Reverse/AbstractSchemaParser.php:34]: same

File: tests/Propel/Tests/Generator/Builder/Om/GeneratedObjectLobTest.php [hunk 3]
```diff
@@ -42,7 +42,7 @@ class GeneratedObjectLobTest extends BookstoreEmptyTestBase
     /**
      * Array of filenames pointing to blob/clob files indexed by the basename.
      *
-     * @var array string[]
+     * @var array<string>
```
Comment (gechetspr) [inline tests/Propel/Tests/Generator/Builder/Om/GeneratedObjectLobTest.php:45]: same

File: src/Propel/Generator/Builder/DataModelBuilder.php (hunk 1 above)
Comment (dereuromark) [reply src/Propel/Generator/Builder/DataModelBuilder.php:57]: probably, I just fixed up some old ones.

**Review by gechetspr — APPROVED** [review]

**Pull request merged**